english/sing-song/vlad.md

```diff
@@ -0,0 +1 @@
+[Thank God. Am so happy to be at the end.] (www.crazyvlad.com)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.